src/heroes/sombra/stealth.opy

```diff
@@ -1,4 +1,4 @@
-#!mainFile "../../dev_main.opy"
+#!mainFile "../main.opy"
```

Comment:
```suggestion
#!mainFile "../dev_main.opy"
```

Reply:
this still compiles normally but when you compile directly in the file it doesnt. The reason to this is cause i simply used a different variation on the file on one of my gamemodes and it uses that instead of the usual ""../dev_main.opy"